lib/viewmodels/members.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/models/models.dart';
+import 'package:seeds/services/http_service.dart';
+
+class MembersModel extends ChangeNotifier {
+  HttpService _http;
+
+  MembersModel({ http }) {
+    _http = http;
+  }
+
+  List<Member> members;
+
+  void fetchMembers() {
+    print("fetch members");
+
+    _http.getMembers().then((result) {
+      print("done - fetch members");
+      members = result;
+      notifyListeners();
+    });
+  }
+
+  void initDependencies(HttpService http) {
+    print("members init dependencies...");
+
+    if (_http == null) {
```

Comment:
why `_http` should never be null as it provided in the constructor so the check is redundant

Reply:
It can be instantiated without providing _http in constructor (it's not a required parameter)
In difference with constructor, initDependencies may be called many times
What actually we check is that this function was called first time - otherwise we don't want to override these values

Reply:
but why you don't use a required parameter as it is actually required here. 
is there a case where you want to create it without the `HttpService`?